src/input.h

```diff
@@ -627,6 +627,24 @@ class input_context
          * (x, y, width, height) would describe an area on the visible window that, if clicked, triggers the action.
          */
 
+        /**
+         * Used to implement mouse "edge scrolling". Returns a
+         * tripoint which is a vector of the resulting "move", i.e.
+         * (0, 0, 0) if the mouse is not at the edge of the screen,
+         * otherwise some (x, y, 0) depending on which edges are hit.
+         *
+         * @param speed The number of steps to scroll.
+         */
+        tripoint mouse_edge_scrolling( const int speed );
+        /**
+         * Returns a numeric representation of the edge scrolling
+         * speed setting, which is the number of steps to scroll under
+         * normal circumstances (e.g. normal zoom level). Callers of
+         * mouse_edge_scrolling() are supposed to pass some function
+         * of this return value as the "speed" parameter.
+         */
+        int mouse_edge_scrolling_speed() const;
```

Comment:
This function shouldn't probably be exposed: it's usage there is like this:
`input::mouse_edge_scrolling( some_multiplier * input::mouse_edge_scrolling_speed() )`, which means it can be boiled down to `input::mouse_edge_scrolling( some_multiplier )`.

Reply:
True! I was imagining that callers might want to use some arbitrary function but that seems like over-engineering.

Reply:
This doesn't seem relevant any more after reverting 71e4a79005480f850fd4deb35ec14b18f40fe070.